File: code/abc214_b.py
s, t = map(int, input().split())
a, b, c, count = 0, 0, 0, 0

for i in range(s + 1):
    for j in range(s + 1):
        for k in range(s + 1):
            if (i + j + k) <= s and (i * j * k) <= t:
                count += 1

print(count)

File: code/abc214_b.rb
def answer(s:, t:)
    count = 0
    for i in 0..s
        for j in 0..s
            for k in 0..s
                if (i+j+k) <= s && (i*j*k) <= t
                    count +=1
                end
            end
        end
    end
    count
end